src/components/Editable.js

```diff
@@ -244,10 +243,10 @@ const Editable = ({ isEditing, thoughtsRanked, contextChain, cursorOffset, showC
       return
     }
 
-    const newNumContext = getContexts(newValue).length
+    const newNumContext = getContexts(store.getState(), newValue).length
     const isNewValueURL = isURL(newValue)
 
-    const contextLengthChange = newNumContext > 0 || newNumContext !== getContexts(oldValueRef.current).length - 1
+    const contextLengthChange = newNumContext > 0 || newNumContext !== getContexts(store.getState(), oldValueRef.current).length - 1
```

Comment:
There are multiple places in this function where store.getState() is being used. We can retrieve it once and use the same instance all over.